uploadTest

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Oh, I didn't recognize this the first time around. Can we handle setup and cleanup for these uploadTest files in the normal running of `01-Driver.t` so they're not actually in the repository? 

In a passing test run it'd be great if the `UPLOAD: { }` section created both of these files, used them in the tests, and then deleted them afterwards. This is perhaps slightly complicated because we want a file in the current working dir for one of the test cases, and not just in a /tmp folder, hummmm.